src/codeu/chat/client/commandline/Chat.java

```diff
@@ -136,8 +156,8 @@ public void invoke(Scanner args) {
     //
     panel.register("u-add", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
-        final String name = args.hasNext() ? args.nextLine().trim() : "";
+      public void invoke(List<String> args) {
+        final String name = args.size() > 0 ? args.get(0) : "";
```

Comment:
@mbassiouny 
can you elaborate on this? would
`final String name = args.size() == 1 ? args.get(0) : "";`
be okay? And instead of having the empty string, you want to print the exception instead, is that right?